src/migrations/1638185287373-AddChainIDToDIDs.ts

```diff
@@ -0,0 +1,91 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddChainIDToDIDs1638185287373 implements MigrationInterface {
+  name = 'AddChainIDToDIDs1638185287373';
+  CHAIN_ID = process.env.CHAIN_ID;
+
+  private checkForChainID(): void {
+    if (!this.CHAIN_ID) {
+      throw new Error(
+        'CHAIN_ID environment variable is not set. Please set it to your chain ID',
+      );
+    }
+  }
+
+  private getUpUpdateSQL(table: string, column: string): string {
+    return `UPDATE public.${table} SET ${column} = 'did:ethr:${this.CHAIN_ID}:' || split_part(${column},':', 3) WHERE ${column} IS NOT NULL;`;
+  }
+
+  private getDownUpdateSQL(table: string, column: string): string {
+    return `UPDATE public.${table} SET ${column} = split_part(${column},':', 1) || ':' || split_part(${column},':', 2) || ':' || split_part(${column},':', 4) WHERE ${column} IS NOT NULL;`;
+  }
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    this.checkForChainID();
+
+    // asset table
+    await queryRunner.query(this.getUpUpdateSQL('asset', 'id'));
+    await queryRunner.query(this.getUpUpdateSQL('asset', 'owner'));
+    await queryRunner.query(this.getUpUpdateSQL('asset', `"offeredTo"`));
+    await queryRunner.query(this.getUpUpdateSQL('asset', `"documentId"`));
+
+    // assets_history table
+    await queryRunner.query(
+      this.getUpUpdateSQL('assets_history', `"emittedBy"`),
+    );
+    await queryRunner.query(
+      this.getUpUpdateSQL('assets_history', `"relatedTo"`),
+    );
+    await queryRunner.query(this.getUpUpdateSQL('assets_history', `"assetId"`));
```

Comment:
The asset contracts aren't really aware of DIDs at the moment. These tables storing DIDs currently? I guess they are 🤔 ?

Reply:
I got the backup of the dev database. I search for all DIDs there and what I found is here.

Reply:
Hmm, ok, let's keep the DIDs for the asset column for now 👍